src/project.c

```diff
@@ -54,7 +54,11 @@
 #include <string.h>
 #include <stdlib.h>
 #include <math.h>                                                              //(5.1.008)
-#include <omp.h>                                                               //(5.1.008)
+#if defined(_OPENMP)
+  #include <omp.h>                                                             //(5.1.008)
+#else
+  int omp_get_num_threads(void) { return 1;}
```

Comment:
Yes, otherwise it is undefined when used bellow https://github.com/OpenWaterAnalytics/Stormwater-Management-Model/pull/85/files#diff-b78ed74c06369709e71f70b882c7ff52R263